src/backend/app/Models/ApiKey.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Models;
+
+use App\Models\Base\BaseModelCentral;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+/**
+ * @property int     $id
+ * @property int     $company_id
+ * @property string  $token_hash
+ * @property bool    $active
+ * @property string  $name
+ * @property string  $last_used_at
+ * @property Company $company
+ */
+class ApiKey extends BaseModelCentral {
```

Comment:
Can you add a doc string here explain what `ApiKey` is used for? It's easy to understand now in context of this PR, but might be confusing if someone looks at it later, i.e. are these ApiKey for MPM? Or Api that MPM uses?

It can be very short, something along the lines: Users can generate ApiKeys for third party platforms to connect to certain MPM APIs. These API's may be used in plugins are secured by the correspond `api-key` auth guard.